### PR TITLE
[new release] decoders, decoders-bencode, decoders-cbor, decoders-ezjsonm, decoders-sexplib and decoders-yojson (0.3.0)

### PR DESCRIPTION
**Pull request by mattjbray**
Elm-inspired decoders for Ocaml

- Project page: <a href="https://github.com/mattjbray/ocaml-decoders">https://github.com/mattjbray/ocaml-decoders</a>
- Documentation: <a href="https://mattjbray.github.io/ocaml-decoders/decoders">https://mattjbray.github.io/ocaml-decoders/decoders</a>

##### CHANGES:

* Add `uncons` primitive (mattjbray/ocaml-decoders#7, @mattjbray)
* Add `Decoders_sexplib` (mattjbray/ocaml-decoders#7, @mattjbray)
* Add `Decoders_cbor` (mattjbray/ocaml-decoders#9, @mattjbray)
* Add `Decoders_bencode` (mattjbray/ocaml-decoders#14, @c-cube)
* Remove `containers` dependency (mattjbray/ocaml-decoders#16, @c-cube)

### CODE REVIEW

File: packages/decoders-bencode/decoders-bencode.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders-bencode/decoders-bencode.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/decoders-cbor/decoders-cbor.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders-cbor/decoders-cbor.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/decoders-ezjsonm/decoders-ezjsonm.0.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders-ezjsonm/decoders-ezjsonm.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/decoders-sexplib/decoders-sexplib.0.3.0/opam [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders-sexplib/decoders-sexplib.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/decoders-yojson/decoders-yojson.0.3.0/opam [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders-yojson/decoders-yojson.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

File: packages/decoders/decoders.0.3.0/opam [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/mattjbray/ocaml-decoders/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/mattjbray/ocaml-decoders.git"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/decoders/decoders.0.3.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

**Comment by mattjbray** [comment]
@kit-ty-kate thanks, will make this change in the source repo too.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**